mise.toml

```diff
@@ -15,7 +15,7 @@ lockfile = false
 disable_backends = ["asdf"]
 
 [tools]
-buf = "1.55.1"
+"ubi:bufbuild/buf" = "1.55.1"
```

Comment:
regular buf, installed from the aqua backend, does not install on windows (it fails with `Failed to install aqua:bufbuild/buf@1.55.1: failed copy: C:\Users\Viam\AppData\Local\mise\downloads\buf\1.55.1\buf-Windows-x86_64.exe -> C:\Users\Viam\AppData\Local\mise\installs\buf\1.55.1\buf/bin/buf.exe`). leaving for posterity